R/kernel.r

```diff
@@ -7,6 +7,9 @@ fromRawJSON <- function(r) {
     fromJSON(s)
 }
 
+#debug <- function(...) cat(sprintf(...), "\n", sep='', file=stderr())
+debug <- function(...){}
```

Comment:
So long as it's easy to point the debug function to proper logging later, I don't mind this as an interim solution. The perfect is the enemy of the good and all that.